lib/connection/statement.js

```diff
@@ -1570,7 +1571,14 @@ function sendSfRequest(statementContext, options, appendQueryParamOnRetry)
     // retry, update the url
     if ((numRetries > 0) && appendQueryParamOnRetry)
     {
-      options.url = Util.url.appendParam(urlOrig, 'retry', true);
+      const retryOption = {
+        url: urlOrig,
+        retryCount: numRetries,
+        retryReason: lastStatusCodeForRetry,
+        includeRetryContext: connectionConfig.getIncludeRetryContext()
```

Comment:
shouldn't it be `getIncludeRetryReason`, not context?

Reply:
Fixed.

Reply:
Fixed.